Runtime/Native/Windows/NativeClient.cs

```diff
@@ -249,50 +249,67 @@ public void SetAttribute(string key, string value)
         /// </summary>
         public IEnumerator SendMinidumpOnStartup(ICollection<string> clientAttachments, IBacktraceApi backtraceApi)
         {
-            // Path to the native crash directory
-            string nativeCrashesDir = Path.Combine(
-                    Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
-                    string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName));
+            // Path to the native crash directories
+            string tempDirectory = string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName);
+            string[] crashDirectories = new string[2] {
+                Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), tempDirectory),
+                Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), tempDirectory)
+            };
+              
+            List<string> nativeCrashesDirs = new List<string>();
+            foreach (string direcotry in crashDirectories)
+            {
+                if (!Directory.Exists(direcotry))
+                {
+                    continue;
+                }
+                nativeCrashesDirs.Add(direcotry);
+            }
 
-            if (string.IsNullOrEmpty(nativeCrashesDir) || !Directory.Exists(nativeCrashesDir))
+            if (nativeCrashesDirs.Count == 0)
             {
                 yield break;
             }
 
-            var attachments = clientAttachments == null
-                ? new List<string>()
-                : new List<string>(clientAttachments);
-
-            var crashDirs = Directory.GetDirectories(nativeCrashesDir);
-
             IDictionary<string, string> attributes = GetScopedAttributes();
```

Comment:
```suggestion
            var attributes = GetScopedAttributes();
```
you can use var here. You should use var whenever it's possible and when you don't need to explicitly say what is the type of something

Reply:
Changes applied.